server/routes/orders.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const mysql = require("mysql");
+
+const pool = mysql.createPool({
+    connectionLimit: 10,
+    host: "localhost",
+    user: "geoffrey",
+    password: "Ya9pQcM3x7BsRqDX",
+    database: "toyzrus"
+});
+
+var router = express.Router();
+
+//serve all orders
+router.get("/", (req, res) => {
+    const allItemsString = 'SELECT * FROM orders;';
```

Comment:
We will need to make routes that only staff/managers can access check to make sure the user is logged in as a user with that level of clearance

Reply:
But that can be for a future commit, just making not of it here